app/core/CocoVueRouter.js

```diff
@@ -0,0 +1,32 @@
+import VueRouter from 'vue-router'
+
+import SchoolAdminDashboard from 'app/views/school-administrator/SchoolAdministratorComponent'
+import SchoolAdminDashboardTeacherListView from 'app/views/school-administrator/teachers/SchoolAdminTeacherListView'
+import SchoolAdminTeacherView from 'app/views/school-administrator/dashboard/SchoolAdminDashboardTeacherView'
+
+import TeacherClassView from 'views/teachers/class/TeacherClassView'
+
+Vue.use(VueRouter)
+
+let vueRouter;
+
+export default function () {
+  if (typeof vueRouter === 'undefined') {
+    vueRouter = new VueRouter({
+      // Routing is currently driven by Backbone Router.  When we migrate away from backbone, switch to history
+      mode: 'abstract',
+
+      routes: [
+        {
+          path: '/school-administrator', component: SchoolAdminDashboard, children: [
+            { path: '', component: SchoolAdminDashboardTeacherListView },
+            { path: 'teacher/:teacherId', component: SchoolAdminTeacherView },
```

Comment:
I understand that `SchoolAdminTeacherView` is similar to `TeacherClassesView`, but with read-only functionality for SAs. Is that correct?

Reply:
Can we rename it to `SchoolAdminTeacherClassesView` if that's correct to make it more intuitive

Reply:
Will definitely improve this name - going to hold off for consensus on naming discussion and then make these changes.